### PR TITLE
README: add 2022 ideas

### PR DESCRIPTION
**Pull request by mistydemeo**
This adds our ideas for 2022, based on the currently open issues. This set of mentors isn't complete: we have a few more available mentors to assign.

@Rylan12, I pencilled you in as a possible mentor based on your comments on the issue, but let me know if you're not available.

@nandahkrishna, I added you for the idea you submitted if that works for you.

### CODE REVIEW

**Comment by Rylan12** [comment]
> @Rylan12, I pencilled you in as a possible mentor based on your comments on the issue, but let me know if you're not available.

I'd need to know more about what being a mentor entails before being able to decide. Is there a webpage with more information?

Also, I am considering participating as a student and I imagine that if I do that, I won't be able to be a mentor as well.

Either way, I plan on being around to help since I'm very familiar with that part of the codebase, so even if I'm not an official mentor I will be able to offer assistance.

File: README.md [hunk 1]
```diff
@@ -31,13 +31,42 @@ https://github.com/blog/2312-how-to-run-a-google-summer-of-code-project-on-githu
...
+#### Homebrew issue: https://github.com/Homebrew/brew/issues/10508
+
+### Autobumping resources
+#### Length: 175 hours
+#### Skills Required: ruby
+#### Mentors: Nanda
+#### Difficulty: Medium
+#### Homebrew issue: https://github.com/Homebrew/gsoc/issues/49
+
+### Autobumping resources
```
Comment (nandahkrishna) [inline README.md:57]: Thanks Misty. Just a small correction:

```suggestion
### Improve behaviour of `HOMEBREW_INSTALL_FROM_API`
```

**Comment by nandahkrishna** [comment]
@Rylan12 I think you may find this page helpful if you haven't already taken a look at it: https://summerofcode.withgoogle.com/rules. It's verbose but provides a good overview of the responsibilities and eligibility criteria for each role (mentor, contributor, etc.).

File: README.md [hunk 2]
```diff
@@ -31,13 +31,42 @@ https://github.com/blog/2312-how-to-run-a-google-summer-of-code-project-on-githu
...
+## 2022 Ideas
 
-### Title
-#### Skills Required: ...
-#### Mentors: ...
-#### Difficulty: ...
+### Deduplicate and cleanup GitHub Actions
+#### Length: 175 hours
+#### Skills Required: shell scripting, ruby
+#### Mentors: Mike, Misty
```
Comment (MikeMcQuaid) [inline README.md:39]: I'm game to provide guidance to mentors but probably not mentor any projects myself this year!

**Review by MikeMcQuaid — COMMENTED** [review]
Looks good bar my mentorship! Thanks @mistydemeo.

**Comment by mistydemeo** [comment]
> Also, I am considering participating as a student and I imagine that if I do that, I won't be able to be a mentor as well.

I think that's right, in which case I'm happy to remove you from the list here.

**Comment by mistydemeo** [comment]
All right, I've removed you, Mike. With two mentors and two admins, I think we meet the minimum they ask for!

**Pull request merged**